executor_manager/executors/docker/git_cache_volume_manager.py

```
@@ -0,0 +1,648 @@
+def volume_exists(volume_name: str) -> bool:
+    """
+    Check if a Docker volume exists.
+
+    Args:
+        volume_name: Name of the volume to check
+
+    Returns:
+        True if volume exists, False otherwise
+    """
+    try:
+        result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)